packages/portfolio-contract/test/evm-chain.exo.test.ts

```diff
@@ -0,0 +1,353 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { makeHeapZone } from '@agoric/zone';
+import { prepareVowTools } from '@agoric/vow';
+import { Far } from '@endo/marshal';
+import { prepareEvmChainKit } from '../src/evm-chain.exo.ts';
+
+// Minimal GMPAccountInfo stub
+const makeAccountInfo = (n: number) => ({
+  namespace: 'eip155' as const,
+  chainName: 'arbitrum' as any, // AxelarChain stub
+  chainId: `eip155:42161:${n}` as any,
```

Comment:
A `chainId` will only have 1 colon.

```suggestion
  chainId: `eip155:42161/${n}` as CaipChainId,
```